### ISSUE TITLE
Prevent screen from turning off while connected

### ISSUE THREAD

**Issue by koush** [koush/vysor.io#669]
from @clubtech
https://github.com/koush/vysor.io/issues/659#issuecomment-743925193

**Comment by clubtech** [comment]
Yes, I am aware of the feature to keep the screen awake when plugged in, but there is also away to do this when the phone is not plugged in as the other two apps I mentioned automatically do this and then release the screen wake back to the default setting once the laptop app disconnects from the phone.

**Comment by koush** [comment]
Yep, just letting you know that it exists as a workaround in the meantime. This should be straightforward to implement.

**Comment by clubtech** [comment]
Thank you very much for implementing this feature, @koush

**Comment by koush** [comment]
relevant adb setting 
https://developer.android.com/reference/android/provider/Settings.Global#STAY_ON_WHILE_PLUGGED_IN

**Comment by clubtech** [comment]
I have no knowledge in programming but I think the way that the Wormwhole app does it is via their android app part. They ask for permissions to “appear on top” (not sure if this permission is related or not though). I think that once the connection is established with their PC client, the android app is the part that’s keeping the screen from sleeping.
Maybe you can have the Vysor android app have the phone keep the screen awake when the connection is established. 
Sounds like that via ADB it’s only possible to do if the phone is plugged in.

**Comment by koush** [comment]
There's a few ways to do it. I think I'll use an always on top method, because that works with wireless too. Just leaving notes for myself in this bug.
